.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,76 @@
+# SPDX-FileCopyrightText: 2022-2023 TII (SSRC) and the Ghaf contributors
+#
```

Comment:
What is the origin of this file? We should probably add another copyright line, respecting the original copyright owner as well as the license from the origin.

Reply:
https://github.com/marketplace/actions/ossf-scorecard-action workflow example/default scorecard.yml